ocean_lib/services/test/test_service.py

```diff
@@ -20,13 +20,15 @@ def test_service():
     """Tests that the get_cost function for ServiceAgreement returns the correct value."""
     ddo_dict = get_sample_ddo()
     service_dict = ddo_dict["services"][0]
+    service_dict["additionalInformation"] = {"message": "Sample DDO"}
```

Comment:
Regarding the "as_dictionary" conversion, I don't think it's necessary. That case, in particular, is for when we have Asset or Service, where we implement specific as_dictionary functions. But here we have a json that comes directly from aquarius, it shouldn't contain any Python objects, just regular json.

Please add a test that also contains lists/nesting inside additionalInformation.